superset-frontend/src/explore/components/controls/AnnotationLayerControl/AnnotationLayer.jsx

```diff
@@ -204,10 +204,25 @@ export default class AnnotationLayer extends React.PureComponent {
     return sources;
   }
 
-  isValidFormula(value, annotationType) {
+  isValidFormula(expression, annotationType) {
```

Comment:
bycatch: let's either rename this to `isInvalidFormula` or flip the return values around (this would be my preference).

Reply:
Done